app/(app)/parking-cards/verification/verification-result.tsx

```diff
@@ -18,13 +18,14 @@ import { VerifyEmailsDto } from '@/modules/backend/openapi-generated'
 type VerificationResultSearchParams = {
   email: string
   status: 'verified' | 'verified-no-cards' | 'link-expired'
+  licencePlates?: string
 }
 
 const VerificationResultPage = () => {
   const t = useTranslation('VerificationResult')
   const queryClient = useQueryClient()
   const navigation = useNavigation()
-  const { email, status } = useLocalSearchParams<VerificationResultSearchParams>()
+  const { email, status, licencePlates } = useLocalSearchParams<VerificationResultSearchParams>()
```

Comment:
Just a nice naming practice - plural indicates that the variable is an array. If we refer to string of multiple items, but it's still a string, it's good to find other name that is not in plural. Usual quick fix is to call it `licencePlatesString`.